tests/build-ai-analyze.test.ts

```
@@ -0,0 +1,125 @@
+
+function createContext() {
+  return {
+    req: {
+      raw: new Request('http://localhost/build/ai_analyze', { method: 'POST' }),
+    },
+    get: vi.fn().mockImplementation((key: string) => key === 'requestId' ? requestId : undefined),
+    json: vi.fn().mockImplementation((data: unknown, status: number) => new Response(JSON.stringify(data), { status, headers: { 'content-type': 'application/json' } })),
+  } as any
+}
+
+function mockBuildRequestRow(row: { app_id: string; status: string; ai_analyzed: boolean } | null) {
```

ESLint

[error] Expected a comma.

(style/member-delimiter-style)